packages/scripts/config/webpack.config.js

```diff
@@ -135,16 +104,16 @@ const config = {
 					test: /[\\/]style(\.module)?\.(sc|sa|c)ss$/,
 					chunks: 'all',
 					enforce: true,
-					automaticNameDelimiter: '-',
+					name( module, chunks, cacheGroupKey ) {
+						return `${ cacheGroupKey }-${ chunks[ 0 ].name }`;
+					},
 				},
 				default: false,
 			},
 		},
 		minimizer: [
 			new TerserPlugin( {
-				cache: true,
 				parallel: true,
-				sourceMap: ! isProduction,
```

Comment:
A similar concern as earlier, both flags were removed from the plugin to use settings from webpack. It would be great to verify if we need to add more changes.